cloudigrade/internal/views.py

```diff
@@ -240,17 +241,25 @@ def recalculate_runs(request):
         if cloud_account_id := data.get("cloud_account_id"):
             cloud_account_id = int(cloud_account_id)
     except (ValueError, TypeError) as e:
-        raise exceptions.ValidationError({"cloud_account_id": e})
+        logger.debug(
+            _("Failed to parse '%s' as a cloud_account_id. %s"), cloud_account_id, e
+        )
+        raise exceptions.ValidationError(
+            {"cloud_account_id": _("Failed to parse given input.")}
```

Comment:
Can we expand these (and the ones below) to include what failed to be parsed? i.e. "Failed to parse '%s' as a cloud_account_id'" basically just dropping the error from the response. Might be more useful.  I'm just not sure if CodeQL would still be happy with those.

Reply:
I had something like that at one point when I was working on this, but then I realized that including the input value and field name in the message are redundant because whoever is calling already knows what they're sending (or they _should!_) and the response already indicates which field is invalid.

For example, making some API calls just now against this branch:

```
❯ http localhost:8000/internal/recalculate_runs/ cloud_account_id="420" since="potato"
HTTP/1.1 400 Bad Request
Allow: OPTIONS, POST
Content-Length: 40
Content-Type: application/json
Date: Mon, 31 Oct 2022 20:22:42 GMT
Referrer-Policy: same-origin
Server: WSGIServer/0.2 CPython/3.9.7
X-CLOUDIGRADE-REQUEST-ID: a2fbee68-cd67-409a-83b7-1538bc441ae2
X-Content-Type-Options: nosniff
X-Frame-Options: DENY

{
    "since": "Failed to parse given input."
}


❯ http localhost:8000/internal/recalculate_runs/ cloud_account_id="hello"
HTTP/1.1 400 Bad Request
Allow: OPTIONS, POST
Content-Length: 51
Content-Type: application/json
Date: Mon, 31 Oct 2022 20:22:47 GMT
Referrer-Policy: same-origin
Server: WSGIServer/0.2 CPython/3.9.7
X-CLOUDIGRADE-REQUEST-ID: 9144ca49-6fa8-4c12-aaef-d13f6bab34a1
X-Content-Type-Options: nosniff
X-Frame-Options: DENY

{
    "cloud_account_id": "Failed to parse given input."
}
```

Reply:
👍